### ISSUE TITLE
FR: Acess to the new "Sequencer: Mute Track in Block" function via keyboard...

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#1072]
first of all: It is a very very cool new feature indeed. 

But Clicking per tracks on a numbers box feels really awkward.
Especially because you have to click, close, listen, reopen again and change. 

This is okay if you know exactly what you want (muted in this case) in the first place.
But to quickly mute this/that/the other one to see what works best is not really 
manageable with the click-box. 

It would be much better if this is accessible via hotkeys like the standart 
mute/solo track functions are as well. (These are just a breeze to work with, because it is so fast.)

I would even go so far to say give an option to change/swap the mute track commands with the mute
track in sequencer block. This way when it comes to arranging in the sequencer track we have the same 
speed in terms of workflow like in the track editor.

Best,
tL.

**Comment by kmatheussen** [comment]
> Especially because you have to click, close, listen, reopen again and change.

I don't think you have to close and reopen though. Just click and listen should be enough.

> This is okay if you know exactly what you want (muted in this case) in the first place.
But to quickly mute this/that/the other one to see what works best is not really
manageable with the click-box.

I don't know if this helps, although I think it should, but you can have more than one click-box open at the same time.

> I would even go so far to say give an option to change/swap the mute track commands with the mute
track in sequencer block. This way when it comes to arranging in the sequencer track we have the same
speed in terms of workflow like in the track editor.

How about a command to apply the mute status of the current block to the current sequencer block?

**Comment by Lykdraft** [comment]
> How about a command to apply the mute status of the current block to the current sequencer block?

Would be perfect.

**Comment by kmatheussen** [comment]
I'm not sure what makes sense here. What are you thinking of when I said "current sequencer block"? I can think of two definitions:

1. A sequencer block that plays the block that is currently in the editor. But in order to identify the current sequencer block, we must play song, and it seems a little bit hectic to do this operation while playing 
song.

2. The sequencer block that was last mouse-clicked. (For this to make sense, the green selection color must probably stay green also after releasing mouse button.)

**Comment by Lykdraft** [comment]
> The sequencer block that was last mouse-clicked. (For this to make sense, the green selection color must probably stay green also after releasing mouse button.)

This. Seems like the default in all Editing Applications. Last selected gets edited. No matter what is played.
(Same in Video Editors, Animation Packages or Music Applications I think...)

**Comment by Lykdraft** [comment]
Probably more difficult to tackle than I thought.... 
I was certain this should have been a very quick thing to do.

**Comment by kmatheussen** [comment]
Well, it's a quite radical change, and I don't want to rush those. As you suggest, the current block configuration should probably use the setting of the current sequencer block, but I have this thought in the back of my head.

**Comment by Lykdraft** [comment]
Actually, I think this can be closed now?

The only additional solution that I see is that the mini-popup window
(where you can click to mute tracks) would borrow the track editors hotkey
for mute when it has focus.

Otherwise, I think it's pretty decent to do via keyboard now already.
